app/controllers/hiring_staff/vacancies/job_specification_controller.rb

```diff
@@ -72,7 +72,7 @@ def save_vacancy_without_validation
   end
 
   def next_step
-    candidate_specification_school_job_path
+    FEATURE_UPLOAD_DOCUMENTS == 'true' ? supporting_documents_school_job_path : candidate_specification_school_job_path
```

Comment:
See https://github.com/DFE-Digital/teacher-vacancy-service/commit/a4cd13097df5d148e59e63407852d22f127a101c re `dotenv` casting this as a string.

Reply:
This has been changed to:

```
UploadDocumentsFeature.enabled? ? ... : ...
```